fix(wsp): Correction de la faute d'orthographe dans la fenêtre ajout des membres

Rotomeca opened the pull request:
FIX: MANTIS 0008791

Rotomeca committed:
fix(wsp): Correction de la faute d'orthographe dans la fenêtre ajout des membres

FIX: MANTIS 0008791

tpayen commented on plugins/mel_workspace/localization/workspace/fr_FR.inc:
```diff
@@ -10,4 +10,5 @@ $labels['home'] = 'Accueil';
 $labels['validation_join_public'] = 'Cet espace est public, cliquez sur "Ok" pour le rejoindre.';
-$labels['no_public_error'] = 'Cet espace n\'est pas publique, veuillez contactez un utilisateur pour vous ajouter. Redirection vers la liste des espaces....';
\ No newline at end of file
+$labels['no_public_error'] = 'Cet espace n\'est pas publique, veuillez contactez un utilisateur pour vous ajouter. Redirection vers la liste des espaces....';
```
public*

tpayen commented on plugins/mel_metapage/localization/fr_FR.inc:
```diff
@@ -504,4 +504,5 @@ $labels['compose-mail'] = 'Composer un email';
-$labels['erase_text'] = 'Effacer la saisie';
\ No newline at end of file
+$labels['erase_text'] = 'Effacer la saisie';
+$labels['add_users_not_added'] = 'Ajouter les utilisateurs qui n\'ont pas été ajoutés ?';
```
Je comprends pas cette phrase x)

Rotomeca replied:
![image](https://github.com/user-attachments/assets/34cf2b3c-046e-4a78-9d0a-d2bb9f8c9a5d)

tpayen replied:
On pourrait mettre "Certains utilisateurs n'ont pas encore été ajoutés à la liste des participants, les ajouter ?"

Rotomeca replied:
Ouais c'est mieux XD J'ai changé =)

tpayen commented on plugins/mel_workspace/localization/workspace/fr_FR.inc:
```diff
@@ -10,4 +10,5 @@ $labels['home'] = 'Accueil';
-$labels['no_public_error'] = 'Cet espace n\'est pas publique, veuillez contactez un utilisateur pour vous ajouter. Redirection vers la liste des espaces....';
\ No newline at end of file
+$labels['no_public_error'] = 'Cet espace n\'est pas publique, veuillez contactez un utilisateur pour vous ajouter. Redirection vers la liste des espaces....';
+$labels['no_in_addressbook'] = 'Les personnes ajoutées ne font pas partie de l\'annuaire...Elles ne sont donc pas ajoutées à l\'espace';
```
Un petit espace après `...` c'est encore mieux !

Rotomeca committed:
fix(txt): Modification pour un texte plus compréhensible

`'Certains utilisateurs n\'ont pas encore été ajoutés à la liste des participants, les ajouter ?'` au lieu de `'Ajouter les utilisateurs qui n\'ont pas été ajoutés ?'`
